tests/epyccel/test_epyccel_modules.py

```diff
@@ -171,6 +170,25 @@ def test_module_7(language):
     mod.reset_c()
     mod.reset_e()
 
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = [
+            pytest.mark.xfail(reason="List wrapper is not implemented yet."),
+            pytest.mark.fortran]
+            ),
+        pytest.param("c", marks = [
+            pytest.mark.xfail(reason="List indexing is not yet supported in C"),
```

Comment:
Please reference the issues that will fix this (this makes it easier to find all xfails that should be removed when you fix the issue as you just need to search for the number)

Reply:
updated cd22d9552fa7c174faa97aaba068fe8ff2acd247